api_tests/npm_wrapper.sh

```diff
@@ -12,5 +12,6 @@ nvm use 11.10.0 || (nvm install 11.10.0 && nvm use 11.10.0);
 cd "${ROOT}/api_tests";
 yarn install;
 
-npm test "./harness.js" \
+npm run build \
```

Comment:
:wrench: Should actually both be `yarn`.
```suggestion
yarn run build \
```